website/sidebars.json

```diff
@@ -6,91 +6,74 @@
       "label": "Quickstart guides",
       "items": ["quickstart_desktop", "quickstart_web", "quickstart_cli"]
     },
-    "user_guide",
-    "config_guide",
     {
       "type": "category",
-      "label": "Config reference",
+      "label": "User guide",
       "items": [
+        "user_guide",
         {
           "type": "autogenerated",
-          "dirName": "config"
+          "dirName": "user_guides"
         }
       ]
     },
+
     {
       "type": "category",
-      "label": "State model reference",
+      "label": "Configuration guide",
       "items": [
+        "config_guide",
         {
           "type": "autogenerated",
-          "dirName": "models"
+          "dirName": "config_guides"
+        },
+
+        {
+          "type": "category",
+          "label": "Configuration schemas",
+          "items": [
+            {
+              "type": "autogenerated",
+              "dirName": "config"
+            }
+          ]
         }
       ]
     },
+
     {
       "type": "category",
       "label": "Developer guides",
       "items": [
         "developer_guide",
-        "devguide_config",
-        "devguide_pluggable_elements",
+        {
+          "type": "autogenerated",
+          "dirName": "developer_guides"
+        },
+
         "urlparams",
-        "api_guide"
-      ]
-    },
-    {
-      "type": "category",
-      "label": "User tutorials",
-      "items": [
-        "tutorials/config_cli",
-        "tutorials/config_gui",
-        "tutorials/plugin_usage",
+        "api_guide",
+
         {
           "type": "category",
-          "label": "Archive",
+          "label": "State model reference",
```

Comment:
If possible, it might be beneficial to have an intro or foreword to these docs. I think they're useful and would be referenced, but individuals looking through the docs for the first time who aren't familiar with MST might get confused.

Reply:
> It might be in the other branch, and this is purely my opinion and doesn't need to be acted upon, but I think the search bar would look better placed before the "looking for JBrowse 1" link rather than after the social links. Imo, this better groups mentally "staying on the site" and "navigating away" (as in, everything prior to "looking for JBrowse 1" has to do with staying on the site, and everything after and including that link navigates you away..but I do think it should stay on the right side overall).
> 
> Here's a preview, and with the width increased a bit to 175px to look a little more natural:
> 
> ![image](https://user-images.githubusercontent.com/83305007/197803226-162909e6-6ec9-454a-820d-6eb996573477.png)

regarding the search bar, this was the default style to have it all the way on the right, but i think you're design looks right. i could look into how to restyle it but how did you make the change in your screenshot?